src/app/docs/text-editor/text-editor-docs.component.html

```diff
@@ -14,9 +14,12 @@
       </p>
     </sky-docs-demo-page-summary>
 
-    <!-- <sky-docs-demo>
-      Demo here.
-    </sky-docs-demo> -->
+    <sky-docs-demo>
+      <sky-text-editor
```

Comment:
I notice that the demo and Stackblitz example do not have a label as indicated in the Anatomy at https://developer.blackbaud.com/skyux-text-editor/docs/text-editor#section-anatomy. We don't list that label as optional, but the fact that you don't have it here suggests to me that it is optional and that we should update the Anatomy guidance to indicate as much. Can you confirm whether the label is optional and whether the menu bar, toolbar, and text entry area are all required?

Reply:
The label is controlled by the consumer's template. It's up to them to provide it. There should be a label in the Stackblitz - it says "Text editor demo" and has an `aria-labelledby` pointing back to it for accessibility. I didn't put it in the demo, because I felt it wasn't necessary but I could easily add that if you think its misleading.

> Can you confirm whether the label is optional and whether the menu bar, toolbar, and text entry area are all required?
The label is optional, as it would be controlled by the consumer's template. It is not an input. Menu bar and toolbar and text area are all provided by default. You cannot turn them off, but you can control what appears in the menu and toolbars by using their respective inputs. Does that make sense?

Reply:
I don't think it's misleading unless we're concerned about consumers leaving the label out. I just wasn't sure if it meant that the Anatomy docs were wrong. I saw the header in Stackblitz, but since the font was different than in the Anatomy graphic, I didn't think that was the label.